tools/test-js-with-node

```diff
@@ -190,6 +192,7 @@ EXEMPT_FILES = make_set(
         "web/src/stream_ui_updates.js",
         "web/src/submessage.js",
         "web/src/subscriber_api.js",
+        "web/src/templates.js",
```

Comment:
We don't want to add any of these files to this list; but I think after my comments below, the main remaining thing you'll need to do is add a simple `dispatch.test.js` test.

Reply:
removed `settings_data.js` and `templates.js` from exempt files as they do not have any changes now after addressing all the feedback. I will create new tests for `server_events_dispatch.js` and `top_left_corner.js` in the updated PR as they have lost test coverage due to new changes.